packages/portal/backend/src/controllers/PersonController.ts

```diff
@@ -256,6 +304,61 @@ export class PersonController {
     //
     //     return {newPeople: newPeople, updatedPeople: updatedPeople};
     // }
+
+    private duplicateDataCheck(data: any[], columnNames: string[]) {
```

Comment:
This also shouldn't be in `PersonController`; the comments even say it would be better in `CSVParser`, or just the `ClasslistAgent`.

Reply:
We went with ClasslistAgent. (See new code)